src/features/generic_events/aggregate/index.js

```diff
@@ -37,12 +37,21 @@ export class Aggregate extends AggregateBase {
 
       this.trackSupportabilityMetrics()
 
+      registerHandler('api-recordCustomEvent', (timestamp, eventType, attributes) => {
+        if (RESERVED_EVENT_TYPES.includes(eventType)) return warn(46)
+        this.addEvent({
+          eventType,
+          timestamp: this.toEpoch(timestamp),
+          ...attributes
+        }, false)
```

Comment:
(asking to increase my understanding)

It looks like this explicitly avoids sending default attributes.  The timestamp requires different treatment so that makes sense.  What about `pageUrl` and `currentUrl`?  Not 100% following why these are not collected for custom events.